### ISSUE TITLE
RPiPlay doesn't support Raspberry Pi OS Bullseye

### ISSUE THREAD

**Issue by a7mad3bdullah** [FD-/RPiPlay#364]
After several tries to get it working, it seems that due to the out-phasing of OpenMax on Bullseye, the video renderer will not work. 

During the installation of RPiPlay, specifically when compling the app command
`cmake ..`
it will thtough an error: 
-- OpenMAX libraries not found, skipping compilation of Raspberry Pi renderer

This issue is mentioned in [issue #296](https://github.com/FD-/RPiPlay/issues/296) and [issue 294](https://github.com/FD-/RPiPlay/issues/294). 

After trying to install [OpenMAX ](https://www.khronos.org/openmaxal)or [ALSA ](https://www.alsa-project.org/wiki/Main_Page)(The new replacement for OpenMax) with no luck to get RPiPlay to work, the only way was to install Raspberry Pi OS Buster on a new SDcard/USB. 

After installing Buster OS, the install of RPiPlay went very smooth without any additional dependencies or tinkering. 

I've tried it on RPI 4 (4GB) and RPI 3B+ using the same SD card, and it worked fine, only needed to re-compile the code
`cmake ..`
then
`make -j`

Here is the OS image to download:
![image](https://github.com/FD-/RPiPlay/assets/44746139/48c29b28-4d59-4e1d-80c7-0dc4b38a6833)

So we might conclude that RPiPlay doesn't support Raspberry Pi OS Bullseye, and may need to be updated to work with ALSA. 
@FD-

**Comment by fduncanh** [comment]
@a7mad3bdullah 
Yes, OpenMax (32bit only) is not supported in RPiOS Bullseye. (OpenMax is for video, ALSA is for sound, so ALSA is NOT
a replacement for OpenMAX)

It is possible (I believe) to install unsupported OpenMax on 32-bit RPiOS Bullseye (search the issues on this site for how to do this).   I believe ALSA is supplied with RPiOS Bullseye (?).

Or use [UxPlay](http://github.com/FDH2/UxPlay) which is a descendant of RPiPlay, that uses GStreamer and Video4Linux2 
(instead of deprecated and abandoned  OpenMax) to access hardware video decoding in the Pi's GPU.

Since uxplay  is packaged  in Debian and Raspian 12 (bookworm)  it is likely that it will be supplied
in  the bookworm update of RPiOS expected in mid-October, to accompany the new R Pi model 5 that is just announced.
